wasm-rpc/src/type_annotated_value.rs

```diff
@@ -72,6 +73,96 @@ pub enum TypeAnnotatedValue {
 }
 
 impl TypeAnnotatedValue {
+    pub fn from_function_parameters(
```

Comment:
This is something I did not see in the first PR but there should not be any JSON code in the `typed_annotated_value` module only in `json`. That's an orthogonal feature to having typeinfo.

You can do `features=["typeinfo"]` and then you don't even have a `serde` dependency. So all these functions constructing `TypedAnnotatedValue` from/to `serde_json::Value` should be in `json`.

Reply:
Sure. Had the same doubt, thanks for confirming

Reply:
Done